src/FsAutoComplete.Core/ParseAndCheckResults.fs

```diff
@@ -25,45 +25,54 @@ type ParseAndCheckResults
 
   let logger = LogProvider.getLoggerByName "ParseAndCheckResults"
 
-  member __.TryGetMethodOverrides (lines: LineStr[]) (pos: pos) = async {
-    // Find the number of `,` in the current signature
-    let commas, _, _ =
```

Comment:
comma calculation was only needed if we actually had a 'noteworthy param location', so I pushed that into the matching branch